contracts/PeeranhaUser.sol

```diff
@@ -62,6 +64,17 @@ contract PeeranhaUser is IPeeranhaUser, Initializable, NativeMetaTransaction, Ac
         userContext.achievementsContainer.peeranhaNFT = IPeeranhaNFT(nftContractAddress);
     }
 
+    /**
+     * @dev Set the admin of role
+     *
+     * Requirements:
+     *
+     * - Must be an existing role.
+     */
+    function setRoleAdmin(bytes32 role, bytes32 adminRole) public onlyRole(PROTOCOL_ADMIN_ROLE) {
```

Comment:
Change `onlyRole(PROTOCOL_ADMIN_ROLE)` to only `onlyRole(DEFAULT_ADMIN_ROLE)`. This is very critical operation.

Also need to add a call `_setRoleAdmin(PROTOCOL_ADMIN_ROLE, BOT_ROLE);` to `__Peeranha_init_unchained` to set admin be default when we redeploy contracts in test environment.

Reply:
You can merge this PR after addressing this comment.